status-im/status-react
# feat(wallet)_: Show buy and receive CTAs on zero balance

fixes #21683

### Summary

This PR 
 - adds `wallet-card` component
 - adds Buy and Receive CTAs above the assets list in the wallet home if the balance is zero

https://github.com/user-attachments/assets/90f3392d-78ab-4913-8749-9fa646ed418b

### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Create a new status profile
- Verify the Buy and Receive CTAs are shown
- Verify on tap of Buy, the Buy bottom sheet opens
- Verify on tap of Receive, the shell share screen with default account QR opens

status: ready
